assets/js/ModuleLibrary.test.js

```diff
@@ -100,8 +112,8 @@ describe('ModuleLibrary', () => {
     
     it('Renders in closed state, without modules', () => {
       expect(wrapper).toMatchSnapshot();
-      // should NOT call getModules (one call from previous test)
-      expect(api.getModules.mock.calls.length).toBe(1);
+      // should NOT call getModules (two calls from previous tests)
+      expect(api.getModules.mock.calls.length).toBe(2);
```

Comment:
This needs to be fixed. Tests should be independent of each other, so you can reorder them without breaking. Move the creation of getModules mock into beforeEach()  so this is always .toBe(1)